docs/proposals/images/machinepool-machines/inframachinepool-delete-replicas.plantuml

```diff
@@ -0,0 +1,33 @@
+@startuml inframachinepool-delete-replicas
```

Comment:
In the below diagram, there are 2 actors involved, providers and core capi. Is there a possibility to distinguish those?

Reply:
Done!